libs/server/src/lib/utils/read-schematic-collections.ts

```diff
@@ -165,17 +165,15 @@ async function readCollectionSchematics(
   collectionName: string,
   collectionJson: any
 ) {
-  const schematicCollection = {
-    name: collectionName,
-    schematics: [] as Schematic[],
-  };
+  const generators = new Set<Schematic>();
+
   try {
     Object.entries(
-      collectionJson.schematics || collectionJson.generators
+      Object.assign({}, collectionJson.schematics, collectionJson.generators)
```

Comment:
I'd like to only do one or the other based on whether the workspace is an Angular workspace or a Nx one.